.buildkite/pipeline.yml

```diff
@@ -46,3 +46,11 @@ steps:
     commands:
       - nix-shell --run 'deploy .#jishui'
     branches: "demo"
+  - label: Push backend image to registry
+    commands:
+      - nix-shell --run 'skopeo copy --dest-creds=serokell-bot:$GITHUB_TOKEN docker-archive:$(nix-build -A packages.x86_64-linux.docker-backend) docker://ghcr.io/serokell/edna-backend:latest'
+    branches: "master"
+  - label: Push frontend image to registry
+    commands:
+      - nix-shell --run 'skopeo copy --dest-creds=serokell-bot:$GITHUB_TOKEN docker-archive:$(nix-build -A packages.x86_64-linux.docker-frontend) docker://ghcr.io/serokell/edna-frontend:latest'
```

Comment:
Is it ok that the actual `GITHUB_TOKEN` is printed in logs?

Reply:
Uhhhh no